gxa/src/main/java/uk/ac/ebi/atlas/home/HomeController.java

```diff
@@ -76,17 +103,27 @@ public String getHome(Model model) {
 
         model.addAllAttributes(latestExperimentsService.fetchLatestExperimentsAttributes());
 
-        model.addAttribute(
-                "speciesList", GSON.toJson(popularSpeciesService.getPopularSpecies(FEATURED_SPECIES)));
-        model.addAttribute(
-                "animalsList", GSON.toJson(popularSpeciesService.getPopularSpecies("animals", FEATURED_SPECIES)));
-        model.addAttribute(
-                "plantsList", GSON.toJson(popularSpeciesService.getPopularSpecies("plants", FEATURED_SPECIES)));
-        model.addAttribute(
-                "fungiList", GSON.toJson(popularSpeciesService.getPopularSpecies("fungi", FEATURED_SPECIES)));
-
         model.addAttribute("speciesPath", ""); // Required by Spring form tag
 
+        JsonArray experimentsjsonArray = experimentInfoListService.getExperimentsJson().get("aaData").getAsJsonArray();
```

Comment:
I think you could get data straight from services and DAO classes, as opposed to parsing the JSON in the backend. If you thinking about it you're going from Java objects -> JSON -> Java objects. It would make your life much easier if you skipped the JSON parsing in the middle! In order to do this, have look at the `listPublicExperiments()` method in the `ExperimentInfoListService`, for example.